pkgs/tools/filesystems/bcache-tools/default.nix

```diff
@@ -20,6 +20,7 @@ stdenv.mkDerivation rec {
     sed -e "/INSTALL.*initramfs\/hook/d" \
         -e "/INSTALL.*initcpio\/install/d" \
         -e "/INSTALL.*dracut\/module-setup.sh/d" \
+        -e "s/pkg-config/${stdenv.cc.targetPrefix}pkg-config/" \
```

Comment:
i think you can also use `$PKG_CONFIG` and the stdenv will set it correctly, for all cases

Reply:
Right, that worked too, thanks :)